openshift/template.yaml

```diff
@@ -43,6 +43,8 @@ objects:
             value: ${REST_VALUE}
           - name: CASSANDRA_SERVICE
             value: ${CASSANDRA_SERVICE}
+          - name: DB_REPLICATION_FACTOR
```

Comment:
Keep uniform prefix while naming parameters. Probably `CASSANDRA_REPLICATION_FACTOR` is better.

Reply:
changed it to CASSANDRA_REPLICATION_FACTOR